cypress/integration/channel/message_spec.js

```diff
@@ -14,28 +14,57 @@ describe('Message', () => {
         cy.visit('/');
 
         // 2. Post a message to force next user message to display a message
-        cy.get('#post_textbox').type('Hello').type('{enter}');
+        cy.post('Hello');
 
         // 3. Login as "user-1" and go to /
         cy.login('user-1');
         cy.visit('/');
 
         // 4. Post message "One"
-        cy.get('#post_textbox').type('One').type('{enter}');
+        cy.post('One');
 
         // * Check profile image is visible
         cy.get('#postListContent > .post.current--user > .post__content > .post__img').last().should('be.visible').find('span > img').should('be.visible');
 
         // 5. Post message "Two"
-        cy.get('#post_textbox').type('Two').type('{enter}');
+        cy.post('Two');
 
         // * Check profile image is not visible
         cy.get('#postListContent > .post.current--user > .post__content > .post__img').last().should('be.visible').should('be.empty');
 
         // 6. Post message "Three"
-        cy.get('#post_textbox').type('Three').type('{enter}');
+        cy.post('Three');
 
         // * Check profile image is not visible
         cy.get('#postListContent > .post.current--user > .post__content > .post__img').last().should('be.visible').should('be.empty');
     });
+
+    it('M14012 Focus move to main input box when a character key is selected', () => {
+        // 1. Login and go to /
+        cy.login('user-1');
+        cy.visit('/');
+
+        // 2. Post message
+        cy.post('message');
+
+        /*eslint max-nested-callbacks: ["error", 3]*/
```

Comment:
👍, please move this on top, just above the `describe`. Similar to what we have on other spec files.